jpegenc/models/rgb2ycbcr/commons.py

```diff
@@ -0,0 +1,29 @@
+#!/bin/python
```

Comment:
Merk btw

you should provide a Makefile per unit test and with targets for three things: native (in the language; Python) simulation, VHDL backend (ghdl) and Verilog backend (iverilog) simulation